ui/src/app/applications/components/resource-details/resource-details.tsx

```diff
@@ -112,6 +114,17 @@ export const ResourceDetails = (props: ResourceDetailsProps) => {
                 }
             ]);
         }
+        if (ExtensionComponent) {
+            tabs.push({
+                title: 'More',
+                key: 'extension',
+                content: (
+                    <div>
+                        <ExtensionComponent node={node} />
```

Comment:
Looks great, just one comment. I think we need to pass the resource tree to the extension here in addition to the node (the Rollout Extension for example needs the tree to render ReplicaSets)

Reply:
ah, good point. Added